kornia/feature/scale_space_detector.py

```diff
@@ -203,17 +204,18 @@ def detect(self, img: Tensor, num_feats: int, mask: Optional[Tensor] = None) ->
             resp_flat_best = resp_flat_best * good_mask.to(dev, dtype)
 
             # Normalize LAFs
-            current_lafs = normalize_laf(current_lafs, octave[:, 0])  # We don`t need # of scale levels, only shape
+            current_lafs *= px_size
```

Comment:
in place won't work with autograd

Reply:
@edgarriba this kind of inplace works usually

Reply:
oh, good to know !!